examples/chip-tool-darwin/templates/tests/partials/test_cluster.zapt

```diff
@@ -91,7 +91,7 @@ class {{filename}}: public TestCommandBridge
       }
 
       // Go on to the next test.
-      WaitForMs(0);
```

Comment:
Down at the bottom on line 278, there is a check for `WaitForMs` that should be updated too, right?

https://github.com/project-chip/connectedhomeip/pull/18383/files#diff-6e2f03fc9524f9d497b08267bf7bfa1806b21f576056458d64752f93e71f1195R278

Reply:
That minCommandTimeout stuff was copied from the Darwin XCTests and is dead code; that partial is never used....  We should kill it.

Reply:
I have removed that code.